core/services/aws/tests/user/updateUser.test.ts

```diff
@@ -124,8 +122,8 @@ describe('updateUserLambda', () => {
       address: undefined
     }
 
-    const mockResponse = 'User updated successfully'
-    mockedUserService.prototype.updateUser.mockResolvedValue(mockResponse)
+    const mockResponse = 'Your Profile info has been updated.'
```

Comment:
same

Reply:
Updated in f98450b71743d5473aa397d4854e7347002cc21c